DeAceroDLabs/marketplace-frontend
# [DMP-679] Update call to product service to get images

## What is done

[[DMP-679] Actualizar llamada a catálogo para obtener imágenens](https://deacero.atlassian.net/browse/DMP-679)

## How is done

1. Updated product props in the api file.
2. Updated Home, Search and ProductDetail views to use the new Image type in their calls.
3. Added a catch on the ImageCard and Carousel components to handle broken urls on images.

## How to test

1. Run the project via `npm start`
2. Check that the app doesn't crash because of a broken image url.

## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)
